tests/utils/test_setup.c

```diff
@@ -27,9 +27,9 @@ void initialize_test(sf_bool debug) {
     if (!test_log_dir) {
         char tmp_dir[4096];
         sf_get_tmp_dir(tmp_dir);
-        snprintf(PERFORMANCE_TEST_RESULTS_PATH, 4096, "%s%s%s", tmp_dir, sep, perf_test_fn);
+        snprintf(PERFORMANCE_TEST_RESULTS_PATH, 5000, "%s%s%s", tmp_dir, sep, perf_test_fn);
```

Comment:
If the max path len is changed to 5000, should the 'tmp_dir' change too? A constant for this length might be better.

Reply:
This is to fix the compile error reported by gcc9. It's smarter than previous version to calculate possible max length of the input and compare with the length of output buffer. The error here was one of the input tmp_dir could be 4096 bytes already, after adding sep and perf_test_fn it could exceed the max length of output buffer which was 4096 bytes.